lib/grpc/server/adapters/cowboy/handler_exception.ex

```diff
@@ -0,0 +1,16 @@
+defmodule GRPC.Server.Adapters.Cowboy.HandlerException do
```

Comment:
I believe this could be a generic module at the adapter level. I know we only have one adapter currently, but this could be useful in the future and would avoid a refactor just to move it around

Reply:
Sure! I can make that refactor (up to end of afternoon)

Reply:
[4b0583e](https://github.com/elixir-grpc/grpc/pull/381/commits/4b0583ebfd942ef4bdb27bc98b8d48530c3a64d0)

Reply:
Let me know what you think. I believe `adapter_extra` can be used to append whatever the adapter needs to report that exception

Reply:
> Let me know what you think. I believe `adapter_extra` can be used to append whatever the adapter needs to report that exception

I agree